app/subscriber/src/features/my-reports/edit/ReportEditForm.tsx

```diff
@@ -78,10 +81,14 @@ export const ReportEditForm = ({ disabled }: IReportEditFormProps) => {
         <ReportEditSummaryForm disabled={disabled} />
       </Show>
       {/* Preview Menu */}
-      <Show visible={active === ReportMainMenuOption.Preview}>
-        <ReportPreview />
+      <Show visible={active === ReportMainMenuOption.View}>
+        <ReportView />
       </Show>
-      <ReportEditActions />
+      {/* Send Menu */}
+      <Show visible={active === ReportMainMenuOption.Send}>
+        <ReportSendForm />
+      </Show>
+      <ReportEditActions disabled={disabled} updateForm={updateForm} />
```

Comment:
Needed `updateForm` to inform the original state of changes.  This issue shows up when submitting the report to be sent out.